src/renderer/extensions/vueNodes/interactions/resize/useNodeResize.ts

```
@@ -2,20 +2,17 @@ import { useEventListener } from '@vueuse/core'
 import { ref } from 'vue'
 
 import type { Point, Size } from '@/renderer/core/layout/types'
+import { layoutStore } from '@/renderer/core/layout/store/layoutStore'
```

ESLint

[error] Unable to resolve path to module '@/renderer/core/layout/store/layoutStore'.

(import-x/no-unresolved)